storage/src/mmr/bitmap.rs

```diff
@@ -706,8 +773,8 @@ mod tests {
             for _ in 0..(Bitmap::<Sha256, SHA256_SIZE>::CHUNK_SIZE * 8 - 1) {
                 bitmap.append(&mut hasher, false).await.unwrap();
                 let newer_root = bitmap.root(&mut hasher).await.unwrap();
-                // root won't change when adding 0s within the same chunk
-                assert_eq!(new_root, newer_root);
+                // root will change when adding 0s within the same chunk
+                assert!(new_root != newer_root);
```

Comment:
nit: `assert_ne`

Reply:
Huh, I somehow had it in my mind that rust didn't yet have an assert_ne! macro!?

Fixed (here & a dozen places elsewhere in the file)